src/components/UserInfo.js

```diff
@@ -1,20 +1,36 @@
 export default class UserInfo {
-  constructor({ profileNameSelector, profileDescriptionSelector }) {
+  constructor(
+    { profileNameSelector, profileDescriptionSelector, profileAvatarSelector },
+    changeAvatarHandler
+  ) {
     this._profileTitle = document.querySelector(profileNameSelector);
     this._profileDescription = document.querySelector(
       profileDescriptionSelector
     );
+    this._profileAvatar = document.querySelector(profileAvatarSelector);
+    this._handleChangeAvatarClick = changeAvatarHandler;
+    this._setEventListeners();
+  }
+
+  _setEventListeners() {
+    this._profileAvatar.addEventListener("click", () => {
+      this._handleChangeAvatarClick(this);
+    });
```

Comment:
Please, delete `_setEventListeners` from `UserInfo`. It's only for setting and getting the user data. There should not be any click handlers

Reply:
done